docs/options.md

```diff
@@ -161,6 +161,8 @@ Whether to include a `<use>` element for each sprite within the generated sprite
 #### `sprite.generate.view` - `false`
 Whether to include a `<view>` element for each sprite within the generated spritemap to allow referencing via [fragment identifiers](https://css-tricks.com/svg-fragment-identifiers-work/). Passing a string will use the value as a postfix for the `id` attribute.
 
+#### `sprite.generate.keepDimensions` - `false`
+Whether to keep each SVG `height` and `width` attributes and add them to the generated symbol. Requires `output.svgo` to be `false` or selectively disabled [SVGO `removeDimensions` plugin](https://svgo.dev/docs/plugins/removeDimensions/).
```

Comment:
>or selectively disabled [SVGO `removeDimensions` plugin]

I think we can do this automagically, like this:

https://github.com/cascornelissen/svg-spritemap-webpack-plugin/blob/85e696bc3e71ffa2af4563386cd5a0afb321f62a/lib/index.js#L172

Reply:
Beautiful idea! Added `removeDimensions` plugin to the `generateSVGOConfig` function ([72570cf](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/pull/219/commits/72570cfdcab83a1e048cc7e528b8ce84c3a8728b)) and changed the docs ([131dc65](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/pull/219/commits/131dc65b9fe46c33afad11f7358ba9e666985ebe))